### PR TITLE
fix: Fix search spaces and capitalization

### PR DESCRIPTION
**Pull request by zhenlei520**
# Description

Fix search spaces and capitalization

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

File: src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs [hunk 1]
```diff
@@ -75,18 +77,38 @@ public class AutoCompleteClient : BaseAutoCompleteClient
         }
     }
 
+    private string GetKeywordByFuzzy(string keyword)
+    {
+        if (_enableMultipleCondition)
+            return string.Join(' ', keyword.Split(' ').Select(CompleteKeyword));
```
Comment (doddgu) [inline src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs:83]:
```csharp
string.Join(' ', keyword.Split(' ').Select(word => $"*{ word.Trim("*") }*"));
```

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs [hunk 2]
```diff
@@ -75,18 +77,38 @@ public class AutoCompleteClient : BaseAutoCompleteClient
...
+
+        return $"\"{keyword}\"";
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs:85]: yes, but disabling spaces doesn't want to treat the content as a phrase?

File: src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs (hunk 1 above)
Comment (zhenlei520) [reply src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs]: Processed

File: src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs [hunk 3]
```diff
@@ -75,18 +77,38 @@ public class AutoCompleteClient : BaseAutoCompleteClient
         }
     }
 
+    private string GetKeywordByFuzzy(string keyword)
```
[the remark this answers is not in the document]
Comment (zhenlei520) [reply src/SearchEngine/Masa.Contrib.SearchEngine.AutoComplete.ElasticSearch/AutoCompleteClient.cs]: Processed

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**